### ISSUE TITLE
Control over GraphqlClient type name?

### ISSUE THREAD

**Issue by njlr** [Zaid-Ajaj/Snowflaqe#3]
Suppose I have a config like this:

```json
{
  "schema": "./introspection.json",
  "queries": "./queries",
  "project": "My.GraphQL.Project",
  "output": "./graphql",
  "target": "fable"
}
```

Then my generated project will have a type like this:

```fsharp
namespace My.GraphQL.Project

// ...

type My.GraphQL.ProjectGraphqlClient(url: string, headers: Header list) =
    new(url: string) = My.GraphQL.ProjectGraphqlClient(url, [ ])

    member _.LatestEntities() =
        async {

// ...
```

The `namespace` is what I expected but the type name of the client is a bit cumbersome.

**Is there a way to control the type name of the client?** 

For example:

```fsharp
namespace My.GraphQL.Project

// ...

type GraphqlClient(url: string, headers: Header list) =
    new(url: string) = My.GraphQL.ProjectGraphqlClient(url, [ ])

    member _.LatestEntities() =
        async {

// ...
```

**Comment by Zaid-Ajaj** [comment]
> Is there a way to control the type name of the client?

Right now there isn't a way but I accept PRs regarding these things :smile: 

Maybe we can add a `overrideClientName` attribute of type string which will do just than or simply remove the project prefix from the `GraphqlClient` type (i.e. adding a config `removeProjectPrefix` of type `bool`)

**Comment by njlr** [comment]
PR: https://github.com/Zaid-Ajaj/Snowflaqe/pull/4

**Comment by Zaid-Ajaj** [comment]
Added and published as of v1.4 :rocket:
